hilti/toolchain/src/compiler/validator.cc

```diff
@@ -260,6 +260,16 @@ struct VisitorPost : visitor::PreOrder, public validator::VisitorMixIn {
             if ( ! n->body() && ! is_hook && ! attrs->has(hilti::attribute::kind::Cxxname) )
                 error(fmt("function '%s' must have a body or be declared with &cxxname", n->id()), n);
         }
+
+        for ( const auto& p : n->ftype()->parameters() ) {
+            if ( p->attributes()->find(hilti::attribute::kind::CxxAnyAsPtr) ) {
+                if ( ! n->attributes()->find(hilti::attribute::kind::Cxxname) )
```

Comment:
just noting that above it checks `if ( auto* attrs = n->attributes() )` but I don't think there's a way for `n->attributes() == nullptr` but this uses `attributes` outside of that check